client/src/pages/ViewProfile.js

```diff
@@ -1,17 +1,27 @@
-import React from "react";
+import React, { useContext } from "react";
 import styled from "styled-components";
+import { ProfileContext } from '../context/ProfileContext';
+import BasicDropDown from '../constant/BasicDropdown';
+import DisplayMyProfile from '../components/DisplayMyProfile';
+import EditMyProfile from '../components/EditMyProfile';
+
 
 const CreateProfile = () => {
+	const { profile, edit }= useContext(ProfileContext);
+
+
 	return (
-		<Container>
-			This is Overview Page
-		</Container>
+		<Screen>
+			{!profile&&<BasicDropDown />}
```

Comment:
I would suggest updating the Routes so that the view profile endpoint takes a profile id in the url, something like  `/viewprofile/123`. That way, you don't have to use the BasicDropDown and you can fetchProfile in a `useEffect()` when the page first loads (like you have with the Home page)